tests/resources/generator/hodgkin_huxley_squid_axon_model_1952/model.external.py

```diff
@@ -89,20 +92,20 @@ def compute_computed_constants(variables):
 
 
 def compute_rates(voi, states, rates, variables, external_variable):
-    variables[0] = external_variable(voi, states, rates, variables, 0)
+    variables[0] = external_variable(voi, states, variables, 0)
```

Comment:
How come this external variable is not seen in the `compute_variables` function?

Reply:
`compute_variables` is for variables that are states/rates-dependent. So... yes, `variables[0]`and `variables[17]` ought to be in `compute_variables` too. Oversight at my end. Will fix it.

Reply:
Done in commit e133304b.